cllc-interfaces/Dynamics-Autorest/Extensions/SpecialEventsExtension.cs

```diff
@@ -162,6 +163,136 @@ public async Task<HttpOperationResponse> AddReferenceWithHttpMessagesAsync(strin
             }
             return _result;
         }
+
+        /// <summary>
+        /// Async version of Get Next Link.  Based on the regular Get and intended for use when paging.
+        /// </summary>
+        /// <param name="nextLink"></param>
+        /// <param name="customHeaders"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<HttpOperationResponse<MicrosoftDynamicsCRMadoxioSpecialeventCollection>> GetNextLinkAsync(string nextLink, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default)
+        {
+            if (nextLink == null)
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, "nextLink");
+            }
+
+            // Tracing
+            bool _shouldTrace = ServiceClientTracing.IsEnabled;
+            string _invocationId = null;
```

Comment:
Do not like this part . Approving for now  as this is critical and need a push to prod